token-swap/js/client/token-swap.js

```diff
@@ -328,23 +380,16 @@ export class TokenSwap {
   }
 
   /**
-   * Swap the tokens in the pool
+   * Swap token A for token B
```

Comment:
The next step of token-swap is to expand to pools, so the function will need to take in the accounts of the swap no matter what -- meaning this can keep the same signature as before, minus the `authority` and `tokenProgramId`.  Going along with that, you can get rid of the second function `swapInverse`

Reply:
Gotcha. Since devs can fetch the token mint accounts with `getSwapInfo`, requiring them to pass in the mint accounts here seems more reasonable.